### PR TITLE
Addressed Tableau submission CI failure

### PR DESCRIPTION
**Pull request by GumpacG**
### Description

Fixed the issues listed below and updated the taco file in the repo.
Made connector more secure by removing username and password in the url as recommended in https://tableau.github.io/connector-plugin-sdk/docs/api-reference.
CI failures include:

- lack of connectionProperties.js file in the connector files

    - Used https://github.com/tableau/connector-plugin-sdk/blob/master/samples/plugins/postgres_jdbc/connectionProperties.js as a reference to add this file.

- lack of use of connectionProperties in the connectionResolver.tdr file
- mismatch version between taco version (1.0) and manifest file version (previously 1.0.0.0, changed to 1.0)

 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: bi-connectors/TableauConnector/src/connectionProperties.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+(function propertiesbuilder(attr) {
+    var props = {};
+    props["user"] = attr[connectionHelper.attributeUsername];
+    props["password"] = attr[connectionHelper.attributePassword];
+
+    if (attr[connectionHelper.attributeSSLMode] == "require") {
+        props["ssl"] = "true";
+        props["sslmode"] = "require";
+    }
```
Comment (Yury-Fridlyand) [inline bi-connectors/TableauConnector/src/connectionProperties.js]: I think `v-trustSelfSigned` and `v-region` should be included here as well, ref:
https://github.com/Bit-Quill/sql-jdbc/blob/c0ce14f5bc867b67ca1ce6641174398ba96b91ad/bi-connectors/TableauConnector/src/connectionBuilder.js#L16-L19

**Review by matthewryanwells — APPROVED** [review]

File: bi-connectors/TableauConnector/src/connectionProperties.js (hunk 1 above)
Comment (GumpacG) [reply bi-connectors/TableauConnector/src/connectionProperties.js]: Made connector more secure by removing username and password in the url as recommended in https://tableau.github.io/connector-plugin-sdk/docs/api-reference. No need for other properties here.

File: bi-connectors/TableauConnector/src/connectionProperties.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+(function propertiesbuilder(attr) {
+    var props = {};
```
Comment (acarbonetto) [inline bi-connectors/TableauConnector/src/connectionProperties.js]: nit: you can do this all with one line:
```
return {
  user: attr[connectionHelper.attributeUsername],
  password: attr[connectionHelper.attributePassword]
};

**Review by acarbonetto — APPROVED** [review]

File: bi-connectors/TableauConnector/src/connectionProperties.js (hunk 1 above)
Comment (acarbonetto) [reply bi-connectors/TableauConnector/src/connectionProperties.js]: Should we raise a follow-up ticket to address this?

Comment (GumpacG) [reply bi-connectors/TableauConnector/src/connectionProperties.js]: No. No further action is needed. Changes are in the this PR.

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**